Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -78,7 +78,7 @@ public class Swerve extends SubsystemBase
   private boolean                  m_locked            = false;
 
   // Path following
-  private int                      m_pathDebug         = 0;    // Debug flag to disable extra ramsete logging calls
```

Comment:
OK to leave on for now, but we will evaluate whether to leave this on during competition. Take note of whether we are getting loop overruns when debug is true.